controllers/foodsController.js

```diff
@@ -47,4 +47,35 @@ const create = (req, res, next) => {
     })
 }
 
-module.exports = {index, show, create}
\ No newline at end of file
+const update = (req, res, next) => {
+
+  let food = req.body.food
+  if (!food) {
+    return res.status(400).send({
+      error: "Unsuccessful update. The format should be { food : { name : foodName , calories : foodCalories } }"
+    })
+  }
+
+  let name = food.name
+  let calories = parseInt(food.calories)
+
+  if(!name || !calories) {
+    return res.status(400).send({
+      error: "Unsuccessful update. Name and calories are required fields. { food : { name : foodName , calories : foodCalories } }"
+    })
+  }
+
+  let id = req.params.id
+  if(!id) {
```

Comment:
There are few steps going on in each of our controller methods. We could consider - once we have everything built out - pulling each of these steps into more organized helper functions. I'm not sure if these helper functions would exist in the controller or if they would be pulled out into the model @corywest @ameseee. That's something that we could talk about at a later time once everything is fully functional.